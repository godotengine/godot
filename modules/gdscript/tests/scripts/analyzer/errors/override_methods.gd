class BaseClass:
    func base_method():
        pass

class ChildClass extends BaseClass:
    @override
    func base_method():
        pass

class NativeMethodOverride extends Node:

    @override
    func _physics_process(delta: float) -> void:
        pass

    @override
    func doesnt_override_anything():
        pass
