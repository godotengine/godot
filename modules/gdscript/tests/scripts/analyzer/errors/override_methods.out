GDTEST_ANALYZER_ERROR
>> ERROR at line 17: The function doesnt_override_anything has the @override annotation, but does not override anything.
